src/Hydraulic/IsothermalCompressible/components.jl

```diff
@@ -357,36 +357,37 @@ Valve with `area` input and discharge coefficient `Cd` defined by https://en.wik
     ODESystem(eqs, t, vars, pars; name, systems)
 end
 
-@component function VolumeBase(; area, dead_volume = 0, Χ1 = 1, Χ2 = 1,
-        name)
-    pars = @parameters begin
+@mtkmodel VolumeBase begin
+
+    @parameters begin
         area = area
         dead_volume = dead_volume
+        x1 = 1
+        x2 = 1
```

Comment:
x1 x2 were `@structural_parameters`. These could be just `@parameters`. However, that would need fixing these scoping here: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/blob/main/src/Hydraulic/IsothermalCompressible/components.jl#L648-L650 in DynamicVolume

Probably adding them as structural_parameters is easier.